### DISCUSSION TITLE
not able to download

### DISCUSSION THREAD

**Discussion by jahanzebweb** [aclap-dev/video-downloadhelper#693] [Help and Q&A]
### Your question

{
  "version": "9.0.1.22",
  "target": "google",
  "channel": "stable",
  "page_url": "xxxxxxxxxxxxxxxxg/s/courses/646a2bf0e4b091d621e4a334/preview",
  "error": {
    "details": "SideDownload error: : [hls @ 000002820549a640] Error when loading first segment 'https://dnqe0v482iyb1.cloudfront.net/spees/w/o/61794ddb0cf2751878772142/v/647780bae4b09ba84482059f/u/61794ddb0cf2751878772143/t/7daa2e1d43a293aa1d00998210957d3d/p/assets/videos/61794ddb0cf2751878772142/2023/05/31/647780bae4b09ba84482059f/hls_2M_000.ts'\r\n[in#0 @ 000002820549a500] Error opening input: Invalid data found when processing input\r\nError opening input file https://dnqe0v482iyb1.cloudfront.net/spees/w/o/61794ddb0cf2751878772142/v/647780bae4b09ba84482059f/u/61794ddb0cf2751878772143/t/7daa2e1d43a293aa1d00998210957d3d/p/assets/videos/61794ddb0cf2751878772142/2023/05/31/647780bae4b09ba84482059f/hls_2M_.m3u8.\r\nError opening input files: Invalid data found when processing input\r\n",
    "downloadable_id": "downloadable_2887111341251315",
    "error": "coapp_failure",
    "id": "coapp_failure_bb8cbd81-77f6-4262-8805-7e7fc2b3114f"
  }
}

### Addon & CoApp version and details

resolve it please

**Comment by GrampaWildWilly** [comment]
You have apparently intentionally obscured the information in the page_url tag.  This is blocking anybody who would help you from helping you.  You need to show that URL properly in order to let somebody else properly investigate this.  On the other hand, I'm guessing from the little bit you have allowed to display that this is some kind of education web site that requires a registration, possibly paid.  In which case, nobody else will be able to help you anyway.

**Comment by Mattmjs** [comment]
It's another of the "unable to open key file" issue. It seems they're all connected to the cloudfront.
Probably another course site where it's behind a password.
